classes/class-wc-qd-invoice-manager.php

```diff
@@ -233,6 +235,34 @@ public function get_payment_method( $order_id ) {
 		}
 		return $method;
 	}
+
+  /**
+   * Get payment processor for Quaderno
+   *
+   * @param $order_id
+   */
+  public function get_payment_processor( $order_id ) {
+    $payment_id = get_post_meta( $order_id, '_payment_method', true );
```

Comment:
We can store any text as payment processor. Why don't we just copy the value provided by WooCommerce?

Reply:
As it is a Quaderno-Woocommerce plugin that is using the Quaderno standard API (not a specific entry point for Woocommerce) I thought that it would make sense to 'translate' from 'Woocommerce terms' to 'Quaderno terms' in this plugin